Jenkinsfile

```diff
@@ -42,6 +42,15 @@ pipeline {
     //     runXmostest("${REPO}", 'tests')
     //   }
     // }
```

Comment:
Do we want to remove these comment-out lines? They're never going to be used again, are they?